src/librustc_mir/transform/const_prop.rs

```diff
@@ -339,12 +339,12 @@ impl<'mir, 'tcx> ConstPropagator<'mir, 'tcx> {
                     // FIXME: implement
                     => {},
 
-                    | Panic { .. }
-                    | BoundsCheck{..}
-                    | Overflow(_)
-                    | OverflowNeg
-                    | DivisionByZero
-                    | RemainderByZero
+                    | Panic(EvalErrorPanic::Panic { .. })
```

Comment:
just use `Panic(_)` instead of listing all of these.

This code here is one of the reasons we want a `Panic` variant